cmd/srcd/daemon/daemon.go

```diff
@@ -110,9 +110,10 @@ func start(workdir string) (*docker.Container, error) {
 		return nil, errors.Wrap(err, "unable to get home dir")
 	}
 
-	tag, hasNew, err := docker.GetCompatibleTag(components.Daemon.Image, cliVersion)
+	cmp := components.Daemon
+	hasNew, err := cmp.RetrieveVersion()
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get compatible daemon version")
+		logrus.Warn("unable to get compatible daemon version from Docker Hub: ", err)
```

Comment:
I wrote this error message but now I see it's kind of misleading. When I read it, it looks like it can't get the daemon itself. Not just check for new version. Can we change it something like "unable to receive information about engine update from Docker Hub"? Or maybe you can find better wording for it.
Thanks!

Reply:
True, it can be read as both getting the version info, or the new image. What about
"failed to list the available daemon versions on Docker Hub: "

Reply:
done, change rebased and force-pushed